tools/.env.template

```diff
@@ -0,0 +1,11 @@
+#Required to use the clone script
```

Comment:
As a general note, it is helpful if template files are not hidden so the user can see that there is a template available.

Reply:
renamed it